envs/xgboost-env.yaml

```diff
@@ -1,7 +1,7 @@
 builder_version: ">=13.0.0"
 
 imported_envs:
-  - common-p10-deps.yaml      #[ppc_arch == "p10"]
+#  - common-p10-deps.yaml      #[ppc_arch == "p10"]
```

Comment:
Can this be removed? Can _common-p10-deps.yaml_ be removed?

Reply:
Done removing references of _common-p10-deps.yaml_.